Felienne/hedy
# [UI IMPROVEMENT] Aligns login and profile UI with signup

**Description**
In this PR we align the UI of all forms across the website with our newest signup UI. We use the same real-time front-end visual feedback on valid input as well as the new rounded input style. This style is changed for the login, profile update and newsletter forms. We remove the old `auth` classes that had become redundant.

**Fixes**
This PR fixes #2025 and fixes #2065

**How to test**
Navigate to the login page and notice the new UI. Verify that the input fields use a front-end validation on the username and password length (aligned with our sign-up requirements). When logged in navigate to profile settings and notice the new profile UI. The keyword language input field is dynamic altered based on user selected language, similar to the signup implementation. Lastly the newsletter form UI is altered to align as well.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
